install

```diff
@@ -84,6 +84,7 @@ fi
 if [[ ! -d "$tmp_dir" ]]; then
     mkdir -p "$tmp_dir" ||
         error "Failed to create temporary directory \"$tmp_dir\""
+    trap 'rm -r "$tmp_dir"' EXIT
```

Comment:
I like this solution. Quick question, is the trap command an `sh` compatible command or is it only `bash` specific? It is to avoid inserting commands that are no shell compatible.

Reply:
Looks like [trap is posix](https://www.man7.org/linux/man-pages/man1/trap.1p.html).